suite-native/accounts/src/components/AddAccountsButton.tsx

```diff
@@ -11,9 +11,12 @@ import {
 } from '@suite-native/navigation';
 import { IconButton } from '@suite-native/atoms';
 import { selectIsPortfolioTrackerDevice } from '@suite-common/wallet-core';
+import { FeatureFlag, useFeatureFlag } from '@suite-native/feature-flags';
 
 export const AddAccountButton = () => {
     const isSelectedDevicePortfolioTracker = useSelector(selectIsPortfolioTrackerDevice);
+    const [isAddCoinAccountEnabled] = useFeatureFlag(FeatureFlag.IsAddCoinAccountEnabled);
+    const [isDeviceConnectEnabled] = useFeatureFlag(FeatureFlag.IsDeviceConnectEnabled);
```

Comment:
Why you don't use this feature flag inside of isAddCoinAccountEnabled, these two are closely tied and there will never be adding of new account without device

Reply:
The way feature flags are written now, I cant use IsDeviceConnectEnabled setting as a base for IsAddCoinAccountEnabled.

And we will probably remove IsAddCoinAccountEnabled flag in next release anyway and only use IsDeviceConnectEnabled.

Reply:
It is there only to hide it for the release and still let qa and us to test it

Reply:
Just simply create hook which will use both of these flags

Reply:
d7ea06eeea